Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h

```diff
@@ -310,6 +310,9 @@ class WebsiteDataStore : public API::ObjectImpl<API::Object::Type::WebsiteDataSt
 #endif
 
 #if USE(CURL)
+    const String& resolvedAlternativeServicesStorageFile() const { return m_resolvedConfiguration->alternativeServicesStorageFile(); }
+    static String defaultAlternativeServicesStorageFile(const String& baseCacheDirectory = nullString());
```

Comment:
Remove `= nullString()`. This is not used.

Reply:
The implementation is aligned with other defaultXXX functions.

https://github.com/WebKit/WebKit/blob/main/Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h#L365-L386